deeprank/learn/NeuralNet.py

```diff
@@ -31,11 +31,9 @@ def __init__(self, data_set, model,
                  model_type='3d', proj2d=0, task='reg',
                  class_weights = None,
                  pretrained_model=None,
-                 chain1='A',
-                 chain2='B',
                  cuda=False, ngpu=0,
                  plot=True,
```

Comment:
Would we have enough data to plot?
Input in our case will be one variant, PSSM, and one structure file. So I would expect one prediction for one variant? 
Plots would make sense we have a dataset of variants.

Reply:
The data_set argument can take one or more hdf5 files as input. Each hdf5 file can contain multiple variants. I could test this also, if you like.

Reply:
ok, we can keep this and wait for data from BioProdict.